angular-tab-trap.js

```diff
@@ -50,10 +50,14 @@
                     focusTrailer.css(buttonStyle);
 
                     children.on('focus', function () {
-                        var focusables = transclusionPoint.find('input');
+                        var potentiallyTabbableChilds = transclusionPoint[0].querySelectorAll('input, button, select, textarea, a[href], *[tabindex]');
 
-                        if (focusables.length) {
-                            focusables[0].focus();
+                        var tababbles = Array.prototype.filter.call(potentiallyTabbableChilds, function(item) {
```

Comment:
Nitpick on white space use, to match style of rest of the code: Please leave a space after function: `function (item)`